starboard/android/shared/platform_configuration/BUILD.gn

```diff
@@ -153,19 +184,23 @@ config("platform_configuration") {
     # Use the static LLVM libc++.
     "-static-libstdc++",
 
+    # TODO(b/371589344#comment2) Revisit if static-libstdc++ is needed.
+    "-Wno-unused-command-line-argument",
+
     # Mimic build/cmake/android.toolchain.cmake in the Android NDK, but
     # force build-id to sha1, so that older lldb versions can still find
     # debugsymbols, see https://github.com/android-ndk/ndk/issues/885
     "-Wl,--build-id=sha1",
     "-Wl,--warn-shared-textrel",
     "-Wl,--fatal-warnings",
+
     "-Wl,--gc-sections",
     "-Wl,-z,nocopyreloc",
 
     # Wrapper synchronizes punch-out video bounds with the UI frame.
     "-Wl,--wrap=eglSwapBuffers",
   ]
-  if (!is_native_target_build) {
+  if (!is_native_target_build && false) {
     ldflags += [
       "-Wl,--wrap=close",
```

Comment:
@kaidokert these wrap flags have issues when compiling the c++ shared library target with the starboard toolchain—unsure what the best way around this is (having libc++ depend on starboard, or ensuring this config isn't applied to that target which could be difficult)